dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -20,11 +20,64 @@
  */
 
 #include "precomp.h"
+#include "iofuncs.h"
+#include "obfuncs.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
 EXTERN_C BOOL PathIsExeW(LPCWSTR lpszPath);
 
+BOOL GetPhysicalFileSize(LPCWSTR PathBuffer, PULARGE_INTEGER Size)
+{
+    UNICODE_STRING FileName;
+    OBJECT_ATTRIBUTES ObjectAttributes;
+    IO_STATUS_BLOCK IoStatusBlock;
+    HANDLE FileHandle;
+    FILE_STANDARD_INFORMATION FileInfo;
+    NTSTATUS Status;
+    WCHAR StdPathBuffer[MAX_PATH] = L"";
+        
+    wcscpy(StdPathBuffer,L"\\??\\");
+    wcscat(StdPathBuffer,PathBuffer);
+    
+    //wcscpy(StdPathBuffer,PathBuffer);
+    RtlInitUnicodeString(&FileName, StdPathBuffer);
```

Comment:
You should use RtlDosPathNameToNtPathName_U instead.

Reply:
good idea !